src/goto-programs/goto_functions.h

```diff
@@ -42,6 +43,12 @@ class goto_functionst
   typedef std::map<irep_idt, goto_functiont> function_mapt;
   function_mapt function_map;
 
+  // For coverage and multi-property
+  // Store and pass the coverage data in incr/kind mode
+  static std::unordered_set<std::string> reached_claims;
```

Comment:
Do we really need these static variables?

Reply:
It is the static member, must be defined and initialized once outside the class definition